ubuntu.json

```diff
@@ -161,7 +161,12 @@
   },
   {
     "alias": "google-cloud",
-    "sourceline": "deb http://packages.cloud.google.com/apt trusty main",
+    "sourceline": "deb http://packages.cloud.google.com/apt cloud-sdk-precise main",
```

Comment:
The existing alias points to Trusty, so that should be preserved as well. If you want to add `google-cloud-trusty`, that fine, but `google-cloud` should not change the definition.